src/components/admin/SearchToolbar.js

```diff
@@ -0,0 +1,77 @@
+// @flow
```

Comment:
Same here, it's not connected to Spoogler flow.

Reply:
I branched `spoogler-flow` out from `search-query`, because I had massive changes of files on the database side and didn't want to wait until that branch was merged. This is why these files ended up here.